amazon/iontest/ion_test_driver.py

```diff
@@ -63,6 +64,8 @@
                                         The order of two implementations matters and the analysis result is based on the
                                         first implementation.
 
+    -s, --specific-desc <description>   Replace a default implementation by the specific description.
```

Comment:
Let's call this option `--replace`.

Reply:
Sure, 

if I called it --replace, do I need a shortcut for it like -r?

I've already used -r and -R.

Reply:
Changed.